ui-tests/src/test/java/org/openmrs/reference/page/ManageUserPage.java

```diff
@@ -23,6 +23,7 @@
     private static final By FIND_USER = By.name("name");
     private static final By USER_SAVED_NOTIFICATION = By.id("openmrs_msg");
     public static String URL_PATH = "/admin/users/users.list";
+    private static final By ADMIN = By.cssSelector("#content > div.box > table > tbody > tr > td:nth-child(1) > a");
```

Comment:
It seems like `ADMIN` is a weird name for this selector. It's only going to be meaningfully `ADMIN` if the string passed to `findUser` is `admin`. Something like `FIRST_RESULT` is a more accurate description.

Also, it seems like the selector could be shortened to `.openmrsSearchTable td:first-of-type > a`.